cf-custom-resources/lib/enable-long-arns.js

```diff
@@ -97,18 +97,16 @@ const enableLongArnFormat = async function () {
  */
 exports.handler = async function (event, context) {
   var responseData = {};
-  var physicalResourceId;
+  const physicalResourceId = event.PhysicalResourceId || event.LogicalResourceId;
 
   try {
     switch (event.RequestType) {
       case "Create":
         await enableLongArnFormat();
-        physicalResourceId = event.PhysicalResourceId;
```

Comment:
Interesting isn't `PhysicalResourceId` always `undefined` for "Create" event? Did we find a bug here while fixing another bug 🤔

Reply:
Yea